docs/applications/monitoring/postinstall.rst

```diff
@@ -0,0 +1,52 @@
+#######################
+Post-Installation Setup
+#######################
+
+The monitoring application requires substantial manual configuration after installation.
+
+Tokens
+======
+
+While the admin token can be supplied from an existing secret, it is necessary to run ``tokenmaker`` to generate both the token that the tasks (which run as cronjobs) will require and the token that the ``telegraf`` and ``telegraf-ds`` scrapers will need in order to send their data to the monitoring database.
+
+To do this:
+
+* Clone the `rubin-influx-tools repository <https://github.com/lsst-sqre/rubin-influx-tools>`__ and change your current working directory to the clone location.
+* Create a new virtualenv to work in.  Activate it.
+* Run ``make init`` to install ``rubin-influx-tools`` into that virtualenv.
+* Export the following environment variables:
+   * ``INFLUXDB_ORG`` should be set to the influx organization, typically ``square``
+   * ``INFLUXDB_TOKEN`` should be set to the admin token value, which can be found in 1Password.
+   * ``INFLUXDB_URL`` should be set to the URL for the InfluxDBv2 server (e.g. ``https://monitoring.lsst.cloud``)
+* Run ``tokenmaker`` and save the tokens, which will be logged to standard output.
+* Clone `Phalanx <https://github.com/lsst-sqre/rubin-influx-tools>`__ and change to the clone location.
+* Create another virtualenv, activate it, and run ``make init`` to install Phalanx.
+* Export the following environment variables:
+   * ``OP_CONNECT_TOKEN`` should be set to the correct 1Password Connect token for the Phalanx environment you're working on.  This can be found in 1Password.
+   * ``VAULT_TOKEN`` should be set to the write token for the Phalanx environment you're working on.  This can also be found in 1Password.
+   * ``VAULT_ADDR`` should be set to your Vault's URL, e.g. ``https://vault.lsst.cloud``
+* In 1Password, find the set of secrets for the Phalanx environment you're working on.  Open the ``monitoring`` secret.  Edit the ``influx-alert-token`` password's value, and change it to the value of "Token for task/alert creation" that was displayed when you ran ``tokenmaker``.  Edit the ``telegraf-token`` password's value, and change it to the value of "Token for remote telegraf bucket writing" that was displayed when you ran ``tokenmaker``.
+* Run ``phalanx secrets audit <environment>``; this should show only ``influx-alert-token`` and ``telegraf-tokens`` with unexpected values.  If anything else is incorrect, fix that first before coming back here.
+* Run ``phalanx secrets sync <environment>``.  This will store the new tokens into Vault.
+* Either wait 15 minutes, or delete the ``monitoring`` secret from the ``monitoring`` namespace in your Phalanx environment.  This will recreate the secret with the new (correct) value.
```

Comment:
Rather than reproducing the documentation for how to do a secrets sync, please instead link to the documentation (`:doc:/admin/update-a-secret` and `:doc:/admin/sync-secrets`) so that it's in only one place and only mention here the things that are specific to this application (such as what secrets to expect to have changed). Otherwise this will get out of date when we make changes.